android/src/main/groovy/com/redhat/ceylon/gradle/android/MyCeylonCompileTask.groovy

```diff
@@ -339,8 +337,7 @@ Depends: ${dependencies}
     args.add(androidRepo.absolutePath)
     args.add("--out")
     args.add(outputRepo.absolutePath)
-    // FIXME: figure that one out
-    args.add("fr.epardaud.testjava.testjava/1")
+    args.add(project.androidCeylon.mainModule)
```

Comment:
Well, in theory this is all auto-detected from the modules defined in `source`. But I agree it's a better temporary step ;)